### ISSUE TITLE
error with at least 1 UA forecast file

### ISSUE THREAD

**Issue by nickreich** [FluSightNetwork/cdc-flusight-ensemble#211]
In running the overall scoring code, we have run into an error in one file. (There might be other errors, but the script stopped when it encountered this one.) I ran the an R function that helps to diagnose problems with forecasts and got the following output:
```
> FluSight::verify_entry_file("model-forecasts/component-models/UA_EpiCos/EW20-2012-UA_EpiCos.csv")
Error in FluSight::verify_probabilities(entry) : 
  ERROR: Missing probabilities detected in HHS Region 1 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 1 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 10 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 10 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 2 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 2 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 3 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 3 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 4 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 4 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 5 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 5 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 6 3 wk ahead.
ERROR: Missing probabilities detected in HHS Region 6 4 wk ahead.
ERROR: Missing probabilities detected in HHS Region 7 3 wk ahead
```
The error seems to be that in some of these target-region combinations there is an `NA` in the `Value` column. For rows that have `Type==Bin`, these must be numeric values and cannot be `NA`. Can you please check over your files (not just this one, but all of them.) and correct this and any other similar errors?

**Comment by hbiegel** [comment]
Thank you for identifying the issue. I believe have fixed the forecast files and used verify_entry_file() on all the forecasts, which results in no errors and only a warning for the 2014/15 season predictions about the week 53 bins. I will commit the updates and submit a pull request.

**Comment by nickreich** [comment]
This was closed by #212
